blog/_posts/2020-05-26-create-your-own-theme-detailed-walkthrough.md

```diff
@@ -0,0 +1,372 @@
+---
+layout: post
+title: Create your own theme in minutes !
+---
+
+In Convos [v4.xx](https://convos.chat/blog/2020/5/14/theming-support-in-4-point-oh.html) it is super easy to customize
+or create a theme.  Follow this detailed walkthrough where I create a **"hack3r"** theme ! 
+
+<!--more-->
+
+
+## Smooth start
+
+You have to connect to your server where convos is running or you can do this tutorial locally : 
```

Comment:
I think "connect" sounds like something you do over the network, which made this confusing to me. You might also want to link to https://convos.chat/doc/faq.html in case the reader does not know what `$CONVOS_HOME` is.

Reply:
I edited a bit this section, probably still not perfect